src/Editor.elm

```diff
@@ -75,10 +76,10 @@ init mts =
 
                 Just ts ->
                     case Helpers.Exporting.Json.Decode.decode ts of
-                        Ok t ->
+                        ( _, Ok t ) ->
                             t
 
-                        Err _ ->
+                        ( _, Err _ ) ->
                             emptyT
```

Comment:
The decoded config should be used, not thrown away. It’d also be good to use `Result.withDefault` on both the tableau and the config part of the `decode`’s result so that if one of them is not decoded, the respective safe default (`emptyT` or `Config.default`, respectively) is used.

This code is used to initialize the editor with the tableau stored in the browser’s localStorage, mainly so that the tableau is not lost completely if the user accidentally reloads the page or closes the browser. Not restoring the config would be quite surprising and inconvenient.